homeassistant/components/lg_soundbar/config_flow.py

```diff
@@ -0,0 +1,76 @@
+"""Config flow to configure the LG Soundbar integration."""
+import temescal
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT, CONF_UNIQUE_ID
+
+from .const import DEFAULT_HOST, DEFAULT_PORT, DOMAIN
+
+DATA_SCHEMA = {
+    vol.Required(CONF_HOST, default=DEFAULT_HOST): str,
+    vol.Required(CONF_PORT, default=DEFAULT_PORT): vol.Coerce(int),
+    vol.Optional(CONF_NAME): str,
```

Comment:
Is it not possible to get the name from the soundbar?

Reply:
it is but was resulting in issues, for me the name was fixed and could not be updated, so i allow the name to be defined, this is Optional and if blank the name from the soundbar is used.
i did want to add a Data_description but that is not suported, i asked on the ddiscored and was told "Data_description only works with selectors, not str or standard voluptuous validators."

Reply:
Users can rename devices and entities via the device info and entity more info pages. There is no need to make this configurable.

Note: You can use a text selector to be able to use `data_description`.